### ISSUE TITLE
Provide a rss-feed with boosts

### ISSUE THREAD

**Issue by Simon0Harms** [tootsuite/mastodon#11290]
At the moment i am using twitter on my homepage to publish news.
It would be great, getting a rss-feed which includes the boosts.

**Comment by yingziwu** [comment]
The atom feed includes the  boosted toots, e.g. https://mastodon.social/users/Gargron.atom
![image](https://user-images.githubusercontent.com/22741471/61170478-296cfb00-a59c-11e9-98d4-6a75a12942e4.png)
![image](https://user-images.githubusercontent.com/22741471/61170481-37bb1700-a59c-11e9-8e7c-d1c84b7bff18.png)

**Comment by jlelse** [comment]
It seems like there is no ATOM feed anymore, just RSS but without boosts. Is there an alternative?

**Comment by sojusnik** [comment]
An ATOM or RSS feed with boosts and replies would be awesome to have!

**Comment by shibattor** [comment]
I would highly appreciate this feature. Currently, the RSS feed is almost useless to me because I see only a part of the someone's activity on Mastodon. That is why I cannot replace Nitter's RSS with Mastodon's.

**Comment by calbasi** [comment]
> I would highly appreciate this feature. Currently, the RSS feed is almost useless to me because I see only a part of the someone's activity on Mastodon. That is why I cannot replace Nitter's RSS with Mastodon's.

I think so. 
Ps. asked here: https://chaos.social/@maralorn/101395328008835661

**Comment by calbasi** [comment]
Maybe we could adapt this code to do the trick, while this issue is open:

https://neil.mckillop.org/mastodon_rss

**Comment by calbasi** [comment]
> Maybe we could adapt this code to do the trick, while this issue is open:
> 
> https://neil.mckillop.org/mastodon_rss

I think this repo is doing it:

https://gitlab.com/idotj/mastodon-embed-feed-timeline

**Comment by calbasi** [comment]
> > Maybe we could adapt this code to do the trick, while this issue is open:
> > https://neil.mckillop.org/mastodon_rss
> 
> I think this repo is doing it:
> 
> https://gitlab.com/idotj/mastodon-embed-feed-timeline

It's on issue #6094

**Comment by woodruffandtheschnibbleofazimuth** [comment]
Anyone got news on this?
RSS without boosts and/or replies are almost useless unfortunately

**Comment by clavelm** [comment]
Same here, I centralize my feeds in my rss aggregator and only rss or atom feeds without boosts or replies is not very useful.
Maybe a way to customize what is in the feed by passing options in the URL?

**Comment by thinkpace** [comment]
Same for me, it would be great to have boosts included in RSS and/or Atom feed.

**Comment by Dennis1000** [comment]
+1, totally useless if boosts aren't included

**Comment by ivarley** [comment]
Yes, I’d also love this as an option, would make it so much more useful.

**Comment by ttarchala** [comment]
+1 from me too

**Comment by ffaf1** [comment]
I would add that a way to have this on or off (like `&boost=yes`) could be even better.

**Comment by paulvierkant** [comment]
+1 please add boosts and responses.

**Comment by rauschma** [comment]
Currently, there are feeds for:

* Their posts (profile URL + `.rss`)
* Their posts and their replies (“posts and replies” URL + `.rss`)

Alas, there is no feed for posts and boosts – which is what I would like to have (if a reply is important, I can boost it).

Interesting boosts are an important part of many accounts. This feature would make such content available to non-Mastodon users via feeds (and hopefully convert them to Mastodon users in the long run).

**Comment by cwalther** [comment]
> * Posts and replies (“posts and replies” URL + `.rss`)

Wow, thanks for mentioning that, I didn’t know that! That’s already a step up! (But yes, boosts would be more important.)

**Comment by ttarchala** [comment]
For full RSS happiness, this is my wishlist

- boosts
- timeline(s)
- context of each toot (e.g. if reply, show along with the toot to which it replied, and the first few replies)
- inline images

**Comment by dblume** [comment]
Not only +1, but I'd be happy with them as separate RSS feeds, no need for one "posts and boosts" feed. We can subscribe to non-intersecting feeds. Doing it this way also wouldn't change behavior of existing RSS readers.

- /users/username.rss (Excludes boosts.)
- /users/username_boosts.rss (where the concatenating character "_" cannot be in username, maybe "+"? Excludes posts.)

**Comment by mnh48** [comment]
I would like this feature as well, I've been looking around on how to get RSS feed with boost included.

**Comment by toelke** [comment]
I have built toelke/mastodon2rss to serve a complete feed of your home timeline.

**Comment by barning** [comment]
I would like to have this because of projects like [Hypebot](https://tldr.nettime.org/@hypebot).

**Comment by Avol-V** [comment]
My own implementation:  
https://github.com/Avol-V/mastodon-rss

You can get separate RSS feeds for everyone you follow. But the channels are created from your home timeline and include all the information that is displayed there, including the boosts.
